pyfcm/fcm.py

```diff
@@ -41,7 +41,7 @@ def notify(
             timeout (int, optional): Set time limit for the request
 
         Returns:
-            dict: Response from FCM server (`multicast_id`, `success`, `failure`, `canonical_ids`, `results`)
+            dict: name (str) - The identifier of the message sent, in the format of projects/*/messages/{message_id}
 
         Raises:
```

Comment:
The new exception raised should be documented. Should not?

Reply:
Good call, I've updated the docstrings for both `fcm.notify` and `baseapi.parse_response` to reflect the 2.x changes.